starport/pkg/cosmosver/detect.go

```diff
@@ -6,9 +6,10 @@ import (
 )
 
 const (
-	cosmosModulePath            = "github.com/cosmos/cosmos-sdk"
-	cosmosModuleMaxLaunchpadTag = "v0.39.99"
-	cosmosModuleStargateTag     = "v0.40.0"
+	cosmosModulePath                  = "github.com/cosmos/cosmos-sdk"
+	cosmosModuleMaxLaunchpadTag       = "v0.39.99"
+	cosmosModuleStargateFortyTag      = "v0.40.0"
+	cosmosModuleStargateFortyThreeTag = "v0.43.0"
```

Comment:
```suggestion
	cosmosModuleStargateFortyThreeTag = "v0.43.0-alpha"
```

To include RC releases and other.